dotnet-desktop-guide/net/wpf/properties/dependency-property-value-precedence.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Dependency property value precedence"
+description: Learn how the precedence of different property-based inputs within the WPF property system determines the effective value of a dependency property.
+ms.date: "09/30/2021"
+dev_langs: 
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "dependency properties [WPF], classes as owners"
+  - "dependency properties [WPF], metadata"
+  - "classes [WPF], owners of dependency properties"
+  - "metadata [WPF], dependency properties"
+---
+<!-- The acrolinx score was 92 on 10/01/2021-->
+
+# Dependency property value precedence
```

Comment:
Don't forget the moniker so we don't have duplicate titles:
```suggestion
# Dependency property value precedence (WPF .NET)
```

Reply:
Noted